Server/src/controllers/getDatasetController.ts

```diff
@@ -22,17 +22,17 @@ export class getDataController {
     async createRequestForData(request: Request, response: Response) {
         let validateData = this.validateInputData(request)
         if (!validateData) {
-            response.status(400).send("Invalid search params entered");
+            response.status(400).json("Invalid search params entered");
         }
         else {
             let requestParams: any = { ...request.query };
             this.processedRequest = requestParams;
             try {
                 const retrieveDataObject = new retrieveData();
                 let arrayOfData = await retrieveDataObject.getArrayOfDatasets(this.processedRequest)
-                return response.status(200).send(arrayOfData);
+                return response.status(200).json(arrayOfData);

```

Comment:
Nice job changing these. We should be using .json on all our responses in the codebase